### PR TITLE
add agent ingress cert issuer docs

### PR DESCRIPTION
**Pull request by Abdiramen**
Resolves https://github.com/ngrok/ngrok-docs/issues/484
Resolves https://github.com/ngrok-private/ngrok/issues/22553

### CODE REVIEW

File: docs/agent/ingress.mdx [hunk 1]
```diff
@@ -83,7 +83,7 @@ configuration file:
 	</TabItem>
 	<TabItem value="go-sdk" label="Go">
 		<CustomIngressGoSdkExample />
-	</TabItem>
+	</TabItCheem>
```
Comment (russorat) [inline docs/agent/ingress.mdx:86]: ?

Comment (russorat) [reply docs/agent/ingress.mdx:86]:
```suggestion
	</TabItem>
```

Comment (Abdiramen) [reply docs/agent/ingress.mdx:86]: huh, not sure how that happened 🤔

File: docs/agent/ingress.mdx [hunk 2]
```diff
@@ -120,6 +120,44 @@ IP addresses. Dedicated IPs that are unique for your account are also available
 at additional cost. [Contact us](mailto:support@ngrok.com?subject=Dedicated+Ingress+IPs) to get in touch for
 additional details.
 
+## Certificate Issuer
+
+While you create a custom agent ingress you'll get the option to chose between
```
Comment (russorat) [inline docs/agent/ingress.mdx:125]:
```suggestion
When you create a custom agent ingress you'll get the option to choose between
```

File: docs/agent/ingress.mdx [hunk 3]
```diff
@@ -120,6 +120,44 @@ IP addresses. Dedicated IPs that are unique for your account are also available
...
+
+The ngrok cert issuer is the default option which the agent trusts
+automatically. It provisions immediately, but may require additional
+configuration if an outbound proxy sits between the agent and ngrok for deep
+traffic inspection, since the proxy won't automaticallly trust the ngrok-issued
+certificate.
+
+### Let's Encrypt Cert Issuer
+
+The Let's Encrypt cert issuer is not automatically trusted by the agent,
```
Comment (russorat) [inline docs/agent/ingress.mdx:142]:
```suggestion
The Let's Encrypt cert issuer is not automatically trusted by the agent and
```

File: docs/agent/ingress.mdx [hunk 4]
```diff
@@ -120,6 +120,44 @@ IP addresses. Dedicated IPs that are unique for your account are also available
...
+requires the `root_cas` option to be set to `host` (or the Let's Encrupt root
+cert) in the configuration file (assuming the host trusts the standard set of
+public CAs). Same goes for any other ngrok SDK, package, or agent.
```
Comment (russorat) [inline docs/agent/ingress.mdx:145]:
```suggestion
public CAs). This also applies to any ngrok agent SDK or SSH Reverse Tunnel agent.
```

**Review by russorat — APPROVED** [review]
LGTM

**Pull request merged**